SemanticClassesWidgetManager has improperly Formatted include

**Describe the bug**
After cloning EasySynth into my Unreal plugins folder, I encountered the following error when trying to compile on Linux:

```[5/38] Compile SemanticClassesWidgetManager.cpp
/home/jess/UnrealEngine/Engine/Plugins/EasySynth-main/Source/EasySynth/Private/Widgets/SemanticClassesWidgetManager.cpp:11:10: fatal error: 'TextureStyles\TextureMappingAsset.h' file not found
   11 | #include "TextureStyles\TextureMappingAsset.h"
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
1 error generated.
```

It looks like the #include directive is using a Windows-style backslash instead of a forward slash. See the offending line [here](https://github.com/ydrive/EasySynth/blob/fd7f100d2ba4f9c539b884f856e6f19c8f4998c7/Source/EasySynth/Private/Widgets/SemanticClassesWidgetManager.cpp#L11). What's odd is that all the other includes in the project use the forward slash.

I manually fixed the line by changing:
`#include "TextureStyles\TextureMappingAsset.h"` to `#include "TextureStyles/TextureMappingAsset.h"` 

After this change, the project compiled successfully on Linux.

**Configuration (if applicable):**
- OS: Linux (Ubuntu 22.04)
- Unreal Engine: 5.5 (built from source)

--- a/Source/EasySynth/Private/Widgets/SemanticClassesWidgetManager.cpp
+++ b/Source/EasySynth/Private/Widgets/SemanticClassesWidgetManager.cpp
@@ -8,7 +8,7 @@
 #include "Widgets/Layout/SUniformGridPanel.h"
 
 #include "EasySynth.h"
-#include "TextureStyles\TextureMappingAsset.h"
+#include "TextureStyles/TextureMappingAsset.h"
 
 
 #define LOCTEXT_NAMESPACE "FSemanticClassesWidgetManager"

